set.mm

```diff
@@ -645874,12 +645897,12 @@ with respect to a linear (1-dimensional) relation.  (Contributed by BJ
     prjspner1.y $e |- ( ph -> Y e. B ) $.
     prjspner1.1 $e |- ( ph -> ( X ` 0 ) =/= .0. ) $.
     prjspner1.2 $e |- ( ph -> ( Y ` 0 ) =/= .0. ) $.
-    $( Vectors whose first coordinate is nonzero are equivalent iff they
-       intersect the (n-1)-th dimensional space (offset 1 from the origin) at
-       the same point.  For example, vectors in 3D space whose ` z ` coordinate
-       is nonzero are equivalent iff they intersect at the plane ` z = 1 ` at
-       the same point (where ` z ` is the first coordinate; also see section
-       header).  (Contributed by SN, 13-Aug-2023.) $)
+    $( Two vectors whose zeroth coordinate is nonzero are equivalent if and
+       only if they have the same representative in the (n-1)-dimensional
+       subspace { x_0 = 1 } .  For example, vectors in 3D space whose ` x `
```

Comment:
**affine** subspace

(Because if you simply write "subspace", it means "vector subspace", that is, a subset that is a vector space for the induced structure, and in particular contains the zero vector.)